src/components/VmDetail/index.js

```diff
@@ -102,10 +98,11 @@ class VmDetail extends Component {
           <dd>{vm.get('fqdn')}</dd>
           <dt><span className='pficon pficon-screen' /> Console</dt>
           <VmConsoles vm={vm} onConsole={onConsole} />
-          <dt><span className='fa fa-database' /> Disks
-            &nbsp;
+          <dt><span className='fa fa-database' /> Disks&nbsp;
             <small>
-              (<a href='#' onClick={onToggleRenderDisks}>{this.state.renderDisks ? 'hide' : 'show'}</a>)
+              ({hasDisks ? (<a href='#' onClick={onToggleRenderDisks}>{this.state.renderDisks ? 'hide' : 'show'}</a>)
```

Comment:
please put the `?` part on its own line as described in http://reactpatterns.com/ (if-else (tidy one-liners))
```javascript
{hasDisks
  ? ...
  : ...
}
```
Additionally, there is no need to wrap the entire (outer) expression in parentheses.